plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -195,14 +196,22 @@ module.exports = new Transformer({
 
 			asset.addIncludedFile(templateFilename);
 
+			if (attributes.import) {
+				fileToImport = await new Promise(resolve =>
```

Comment:
I decided to go with the current approach to avoid requiring extra files. We're already using `marked` on line 148 for transforming markdown to HTML.  	

The same result can be achieved by using `posthtml`, but it involves installing plug-ins that use either `marked` or `markdown-it` under the hood. 

[posthtml-markdownit](https://github.com/posthtml/posthtml-markdownit).
```
const posthtml = require('posthtml')
const markdown = require('posthtml-markdownit')

importFileParsedCode = await posthtml([markdown()])
					.process(`<div class="markdown"><markdown src=${attributes.import}></markdown></div>`)
					.then(result => {
						return result.html;
					});
```